.github/workflows/docbuild.yml

```diff
@@ -6,6 +6,12 @@ on:
     branches:
       - main
       - 'v*-branch'
+    paths:
+      - '*.rst'
+      - 'doc/**'
+      - 'include/**'
+      - 'west.yml'
+      - '**/Kconfig'
```

Comment:
also include the workflow itself, and `scripts/requirements-doc.txt`

Reply:
Thanks. Added.